### DISCUSSION TITLE
böyle bir hata alıyorum ne yapmam lazım?

### DISCUSSION THREAD

**Discussion by tifi00** [19/classroom#21] [Anonim] [answered]
![Adsız](https://user-images.githubusercontent.com/98906910/158033488-af56a8d3-fbbf-4923-a968-3fa7023fd959.png)

**Comment by roktas** [comment, marked as answer]
Yayınlanan dokümanları okudun mu?  Ekran kaydını dikkatli izle: https://classroom.alaturka.dev/guide/#vs-code-icinden-wsl-e-erisim

**Comment by roktas** [comment]
Sorunu anlayamamışsan veya hiç bir şekilde çözemiyorsan, ödevi tarayıcıda yap.
